packages/@orchidui-vue/src/Form/Input/OcInput.vue

```diff
@@ -76,6 +76,18 @@ const props = defineProps({
     type: Boolean,
     default: true,
   },
+  inputMode: {
```

Comment:
hi @FaresG why not use inputType ? cc @mykhailo-diakovych

Reply:
I can add that aswel (once for all)
But for my case, I need inputmode attribute.

Reply:
browser support https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode

Reply:
@kewcoder inputmode is supported on iOS Safari. To show Numpad / Keyboard based on inputType value.
I believe it will be useful for mobile UX.

Reply:
@kewcoder Can we benefit from inputmode? for mobile devices.